app/ui/src/app/integration/import-export/integration-import-export.component.scss

```diff
@@ -0,0 +1,24 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: visible;
+}
+
+ul {
+  flex: 0 0 auto;
+  padding-left: 20px;
+  padding-right: 20px;
+
+  &.toolbar-pf {
+    background: inherit;
+
+    li {
+      &.active {
+        cursor: default;
+      }
+
+      cursor: pointer;
+    }
+  }
+}
```

Comment:
All the `<ul>` styling here has been thoroughly duplicated all over the project and in order to prevent more duplication we now have a global classname named `.syn-tabs` that takes care of that for the entire project. FYI it is available at `/scss/components/_tabbed-nav.scss` and you do not need to import `syndesis-sass` to use it in your HTML because it is already part of the CSS classnames bndle.

Please avoid SASS duplication.